### ISSUE TITLE
[BUG] Linux build is broken because of missing `python-appimage` dependency

### ISSUE THREAD

**Issue by jonathanperret** [AllYarnsAreBeautiful/ayab-desktop#721]
I hit this accidentally building a test release on my fork but it will happen next time we try to build a release here: https://github.com/jonathanperret/ayab-desktop/actions/runs/11542883130/job/32126632072

It appears the `https://github.com/niess/python-appimage/releases/download/python3.11/python3.11.9-cp311-cp311-manylinux_2_28_x86_64.AppImage` link is no longer valid. Somehow the https://github.com/niess/python-appimage project is pulling artifacts from their releases.

**Comment by dl1com** [comment]
It seems they updated from 3.11.9 to 3.11.10.
So altering our download should fix this.

But unfortunately this will break over and over again in the future. Don't know what would be a sensible way to be resilient against this (other than keeping a "local" copy of their artifact).
Perhaps using a wildcard in the Downloadlink? ("3.11.*")

**Comment by jonathanperret** [comment]
Yes that's unfortunate. I plan on opening an issue on https://github.com/niess/python-appimage to get clarity on whether this removing of release artifacts is deliberate.

**Comment by jonathanperret** [comment]
Reopening this as it happened again as expected.

In the meantime, I opened [niess/python-appimage#88](https://redirect.github.com/niess/python-appimage/issues/88) to signal the problem to the `python-appimage` project.

Over in that issue I documented a workaround that involved rebuilding the necessary AppImage from a persistent source (the `manylinux` Docker image repository).
